gax-java/gax/src/main/java/com/google/api/gax/batching/BatcherImpl.java

```diff
@@ -457,6 +494,17 @@ void add(
       totalThrottledTimeMs += throttledTimeMs;
     }
 
+    void setOperation(@Nonnull ApiFuture<ResponseT> operation) {
```

Comment:
I feel `operation` might be a too generic name, can we rename it to something more specific? Maybe something like `responseFuture`?